packages/expo-updates/ios/EXUpdates/EXUpdatesAppController.m

```diff
@@ -227,6 +226,26 @@ - (BOOL)isUsingEmbeddedAssets
 
 # pragma mark - internal
 
+- (void)_finishWithSuccess:(BOOL)success
+{
+  if (_delegate) {
+    [EXUpdatesUtils runBlockOnMainThread:^{
+      [self->_delegate appController:self didStartWithSuccess:success];
+      id splashScreenService = [UMModuleRegistryProvider getSingletonModuleForClass:NSClassFromString(@"EXSplashScreenService")];
```

Comment:
I don't think Updates should know about SplashScreen. At this point it is better to make it a hard dependency -- the dependency is there anyway so it's better to be explicit than implicit.

Instead, the code that initiates the check for updates (the app delegate?) likely should be responsible for calling into the splash screen.

Reply:
OK, that makes sense.

@bbarthec what do you think about having `expo-splash-screen-command` take care of this if it detects expo-updates is installed & setup in AppDelegate already?

I will also add a note to the expo-updates README in case people install them in the opposite order (splash screen, then updates).

Reply:
Yeah, that makes sense to me to make `@expo/configure-splash-screen` a little more aware of it's surrounding (e.g. `expo-updates` in this case). 👍 
May I ask you to provide some manual instructions for achieving this scenario and I'll take care of automating it via `@expo/configure-splash-screen`?

Reply:
Yes, done in e5cc03bf5ec3cdd50f059301c247b690173ab181!